packages/messaging/src/controllers/controller-interface.ts

```diff
@@ -128,7 +128,7 @@ export abstract class ControllerInterface {
     );
     if (isTokenValid) {
       const now = Date.now();
-      if (now < tokenDetails.createTime + TOKEN_EXPIRATION_MILLIS) {
+      if (now < tokenDetails.createTime! + TOKEN_EXPIRATION_MILLIS) {
```

Comment:
token details should be our own internal data structure so either the type is wrong or we should handle this being null.

If you want to just keep the current behavior can we add a TODO comment to handle if it's null.

Reply:
TokenDetails without createTime is used throughout the codebase. CreateTime gets added right before it's saved to IndexedDB. We talked about it with @pinarx offline and decided to add createTime (and some other missing properties) as soon as the TokenDetails object is created. This will remove the need to make these types optional.

I didn't add a TODO because I already have a PR that fixes this, but I can't submit it before my other PRs are merged.